Author: aszlig <aszlig@nix.build>
Date:   Wed Apr 11 17:46:08 2018 +0200

    Do not set a time out for mocha tests
    
    Within Nix builds there already is a timeout, so there is really no need
    to enforce it by mocha as well.
    
    Signed-off-by: aszlig <aszlig@nix.build>
    Filename: mocha-no-timeout.patch

diff --git a/test/mocha.opts b/test/mocha.opts
index 28345d5049..093c26c6aa 100644
--- a/test/mocha.opts
+++ b/test/mocha.opts
@@ -1,6 +1,6 @@
 --colors
 --reporter spec
---timeout 8000
+--no-timeouts
 --check-leaks
 --globals io
 --require babel-register
